.eslintrc.js

```diff
@@ -123,6 +123,8 @@ module.exports = {
         '@typescript-eslint/unbound-method': 0,
 
         'jest/valid-title': [0, { ignoreTypeOfDescribeName: true }],
+        'max-classes-per-file': 0,
+        'class-methods-use-this': 0,
```

Comment:
This is only for tests files?

Reply:
Yep. This applies to `files: ['**/*.spec.ts', 'test/**']`